lib/debezium/schema.go

```diff
@@ -53,13 +53,21 @@ const (
 	Map     FieldType = "map"
 )
 
+type Item struct {
+	Type         FieldType             `json:"type"`
+	Optional     bool                  `json:"optional"`
+	DebeziumType SupportedDebeziumType `json:"name"`
+}
+
 type Field struct {
 	Type         FieldType             `json:"type"`
 	Optional     bool                  `json:"optional"`
 	Default      any                   `json:"default"`
 	FieldName    string                `json:"field"`
 	DebeziumType SupportedDebeziumType `json:"name"`
 	Parameters   map[string]any        `json:"parameters"`
+	// [Items] is only populated if the literal type is an array.
+	Items Item `json:"items"`
```

Comment:
`Items` being plural is a little confusing here, maybe we could call this `ItemsMetadata`?

Reply:
Sg